python/cuml/test/test_dbscan.py

```diff
@@ -14,125 +14,80 @@
 #
 
 import pytest
-from cuml import DBSCAN as cuDBSCAN
 from cuml.test.utils import get_handle
+from cuml import DBSCAN as cuDBSCAN
```

Comment:
It would be nice if we had a consistent standard when importing cuml and sklearn versions, whether to import like `cuDBSCAN` or `DBSCAN_cuml`. You use the latter style for UMAP later in this change. Maybe just pick whichever is most common now?